bundles/org.eclipse.passage.lbc.json/src/org/eclipse/passage/lbc/json/BoundLicenseDeserializer.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lbc.json;
+
+import java.io.IOException;
+
+import org.eclipse.passage.lbc.internal.api.persistence.BoundLicense;
+import org.eclipse.passage.lbc.internal.base.BaseBoundLicense;
+import org.eclipse.passage.lbc.internal.base.ConditionIdentifier;
+import org.eclipse.passage.lbc.internal.base.LicenseCapacity;
+import org.eclipse.passage.lbc.internal.base.LicenseTaken;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+public final class BoundLicenseDeserializer extends StdDeserializer<BoundLicense> {
+
+	/**
+	 * 
```

Comment:
This javadoc comment is a nice place to explain and excuse `static` field declaration as it's a framework-driven demand.